src/components/Chart.tsx

```diff
@@ -75,12 +75,15 @@ export function Chart(props: ChartProps) {
   const createGroupHandlers = (d: CircleOrRectHiearchyNode) =>
     isBlob(d.data)
       ? {
-        onClick: () => setClickedBlob(d.data as HydratedGitBlobObject),
+        onClick: () => setClickedObject(d.data as HydratedGitBlobObject),
         onMouseOver: () => setHoveredBlob(d.data as HydratedGitBlobObject),
         onMouseOut: () => setHoveredBlob(null),
       }
       : {
-        onClick: () => setClickedBlob(null),
+        onClick: () => {
+          setClickedObject(d.data as HydratedGitTreeObject)
+          setPath(d.data.path)
```

Comment:
@tjomson Since clicking an object changes the path, could we remove the path context and derive the path from the clicked blob instead? This way, we reduce the amount of state and it also fixes that closing the details pane, doesn't reset the path.

Reply:
Emil and Kristoffer did not like that interaction, but it was pretty easy to make, if we want to do it like that at some point